src/index.css

```diff
@@ -0,0 +1,6 @@
+body {
```

Comment:
If we create a div around our home page then we can directly apply these style to this.
We should always try to apply css to our container and not the whole body.

Reply:
But ma'am in index.js file where it creates root App component with class name root, it defaultly takes some style and margin for the root. and to clear this I used index.css file. Bt I've now converted it into scss